### PR TITLE
Update Patterns to Designs in Delete Modal and Toaster

### LINKED ISSUE #5815: [UI] Update Patterns to Designs in Delete Modal and Toaster
#### Current Behavior
Patterns are now updated to Designs, delete modal and snackbar is still showing patterns
Screenshot from 2022-07-14 00-08-45
Screenshot from 2022-07-14 00-08-55

#### Expected Behavior
Change Patterns to Designs in both places.

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by kamalbuilds**
Signed-off-by: Kamal <geniusamansingh@gmail.com>

**Description**
Update Patterns to Designs in Delete Modal including title and Toaster
This PR fixes #5815

**Notes for Reviewers**

image

Screenshot from 2022-07-27 14-50-38

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
Yes, I signed my commits.

### CODE REVIEW

**Comment by legendarykamal** [comment]
@Chadha93 pls review

**Comment by leecalcote** [comment]
@legendarykamal it'd be great if you can add the name (or names) of the designs being deleted. The count is good. The count plus design name is even better.

**Comment by legendarykamal** [comment]
Roger @leecalcote , done

image

I will update my commits soon

**Comment by warunicorn19** [comment]
Let's have a whitespace after the comma please.

**Comment by legendarykamal** [comment]
ok @warunicorn19 seems good now

image

additional ss
image

**Review by Chadha93 — APPROVED** [review]
Good work @legendarykamal, let's merge after the workflows are complete.

**Comment by legendarykamal** [comment]
thanks @Chadha93 Yes merging gives a satisfaction feel :smile:

**Comment by warunicorn19** [comment]
I dont know if I'm missing any comments here, did we decide to go back to patterns from designs??

**Comment by Chadha93** [comment]
> I dont know if I'm missing any comments here, did we decide to go back to patterns from designs??

No, The issue is about switching from Patterns to Designs. The first ss are correct. @legendarykamal please update the new ss also.

File: ui/components/MesheryPatterns.js [hunk 1]
```diff
@@ -658,11 +658,12 @@ function MesheryPatterns({
...
   });
 
-  async function showModal(count) {
+  async function showModal(count, patterns) {
+    console.log("patterns to be deleted", count, patterns);
     let response = await modalRef.current.show({
       title : `Delete ${count ? count : ""} Pattern${count > 1 ? "s" : ''}?`,
 
-      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this'}  ${count ? count : ""}  pattern${count > 1 ? "s" : ''}?`,
+      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this'} ${patterns} pattern${count > 1 ? "s" : ''}?`,
```
Comment (warunicorn19) [inline ui/components/MesheryPatterns.js:666]:
```suggestion
      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this'} ${patterns} pattern${count > 1 ? "s" : ''}?`,
```
There's no need for "this" and "these", you can change it to "the  (pattern-name) design or designs"

Comment (legendarykamal) [reply ui/components/MesheryPatterns.js:666]: final updated ss 

image

**Comment by legendarykamal** [comment]
thanks for the feedback @warunicorn19 I have updated :+1:

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thank you for this @legendarykamal

**Pull request merged**